roles/satellite-clone/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # more info about these variables can be found in satellite-clone-vars.sample.yml in the root directory of this project
-satellite_version: 6.2
+satellite_version: 6.1
```

Comment:
satellite_version defaults to 6.1 and is overridden to 6.2 if the satellite-answers file exists. Do you have a particular concern about doing it this way?